ui/app/routes/vault/cluster/access/identity/show.js

```diff
@@ -20,11 +20,6 @@ export default Route.extend({
     // if the record is in the store use that
     let model = this.store.peekRecord(modelType, params.item_id);
 
-    // if we don't have creationTime, we only have a partial model so reload
-    if (model && !model.get('creationTime')) {
```

Comment:
Is this block part of this fix or a different bug? If part of the other bug we should describe it in the PR description

Reply:
I'm actually going to remove it.  It's apart of a larger issue that has several other bugs attached to it. I've got the other bugs documented and will add this to the list.  Good call.